src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -95,10 +107,18 @@ public void satisfiesEqualsContract() throws IOException {
                 )
             )
         );
-        MatcherAssert.assertThat(
-            new RqMultipart.Base(req),
-            Matchers.equalTo(new RqMultipart.Base(req))
-        );
+        final RqMultipart.Base reqbaseone = new RqMultipart.Base(req);
```

Comment:
@raphaelln please avoid compound names: http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html